recipes/dateutil/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version="2.5.2" %}
+
+package:
+  name: python-dateutil
+  version: {{ version }}
+
+source:
+  fn: python-dateutil-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/p/python-dateutil/python-dateutil-{{ version }}.tar.gz
+  md5: eafe168e8f404bf384514f5116eedbb6
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - dateutil
+    - dateutil.rrule
+
+about:
+  home: https://dateutil.readthedocs.org/
+  license: BSD
```

Comment:
Added an issue ( https://github.com/conda-forge/dateutil-feedstock/issues/1 ) to the feedstock.